### ISSUE TITLE
Error  require 'fast_jsonapi/instrumentation/skylight'  in initializer.

### ISSUE THREAD

**Issue by jnjcub** [Netflix/fast_jsonapi#202]
require 'fast_jsonapi/instrumentation/skylight'  in initializer.

>>> rails s
/gems/fast_jsonapi-1.1.1/lib/fast_jsonapi/instrumentation/skylight/normalizers/serializable_hash.rb:8:in `<module:Normalizers>': uninitialized constant FastJsonapi:
:Instrumentation::Skylight::Normalizers::Normalizer (NameError)

skylight-ruby version 1.5.1
Using rails 5.1

**Comment by rromanchuk** [comment]
can confirm 

```
	 1: from /Users/ryan/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/active_support.rb:8:in `without_bootsnap_cache'
/Users/ryan/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/active_support.rb:53:in `block in load_missing_constant': uninitialized constant FastJsonapi::Instrumentation::Skylight::Normalizers::Normalizer (NameError)
```

**Comment by hmcfletch** [comment]
Looks like the normalizers got moved in Skylight v1.5. Will need to do some checks against Skylight version.

I believe the work around would be to restrict Skylight to v1.4.x.

https://github.com/skylightio/skylight-ruby/commit/a49aa07878cba6223e17822e5e093b29e3717db1#diff-7f06462ac8c8f6b4138397d53ac152f6

**Comment by shishirmk** [comment]
This should be released in version 1.2 of the gem
